bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -68,8 +68,6 @@ get_size_for_available_space = (use_width, use_height, client_width, client_heig
 
 # TODO (bev) PlotView should not be a RendererView
 class PlotView extends Renderer.View
-
-  className: "bk-plot"
```

Comment:
So the `className: "bk-plot"` was the reason the tool CSS didn't work. The plot view className is how the rendered div gets a class name, not the plot_div.html that you modified (I'm not sure that template is ever used tbh)